parm/config/config.prep

```diff
@@ -9,6 +9,7 @@ echo "BEGIN: config.prep"
 . $EXPDIR/config.resources prep
 
 export DO_MAKEPREPBUFR="YES"   # if NO, will copy prepbufr from globaldump
+export cdate10=${PDY}${cyc}
```

Comment:
Is PDY and cyc set at the job level or in `prep.sh` with `setPDY`?

Reply:
When run via rocoto both `PDY` and `cyc` are exported at submission time. They are two of the known environment variables for each job at runtime. Example from the rocoto xml (specifically the prep task block):

```
        <envar><name>PDY</name><value><cyclestr>@Y@m@d</cyclestr></value></envar>
        <envar><name>cyc</name><value><cyclestr>@H</cyclestr></value></envar>
```

I considered using `CDATE` as well but I know we will be moving away from `CDATE` in the near future and replacing all `CDATE` instances with `$PDY$cyc` anyway.

In operations via ecflow setpdy.sh is used to set `PDY` and I see `cyc` set at submission, although in ops `cdate10` is set from `ncepdate` and the override I added into the obsproc scripts won't be used.

Right now setpdy.sh doesn't work outside of the operational environment since it uses a file that only exists in production. Here is what happens when setpdy.sh is invoked on Orion with `export COMROOT=/work/noaa/global/glopara/com` set from config.base (which I set for other uses and to avoid errors at runtime by not having `COMROOT` set):
```
0 + setpdy.sh
sed: can't read /work/noaa/global/glopara/com/date/t18z: No such file or directory
```

I considered and tried using setpdy.sh in the prep jobs on Orion but it would require setting up something similar to the `com/date/tCCz` files like is used in ops. For our development needs it's much simpler to set `cdate10` ourselves and have it used by the `cdate10` variable override in the obsproc script.

The setpdy.sh issue is related to general issues with `prod_util` outside of the operational environment and the various utilities within it being used elsewhere.

@lgannoaa When running the GFS in ecflow in the dev environment, how do you set `PDY` and `cyc` at runtime? Thanks!

Reply:
@lgannoaa Ok, thanks for that information!

@aerorahul We have `PDY` and `cyc` set at runtime from above in both the rocoto and ecflow setups.